app/models/duty_option_result.rb

```diff
@@ -13,4 +13,18 @@ class DutyOptionResult
   attribute :warning_text
   attribute :order_number
   attribute :geographical_area_description
+
+  def footnote
+    (super + footnote_suffix).html_safe
```

Comment:
Does footnote_suffix have html content in? If not you could use safe_join which will auto escape?

```ruby
safe_join [super, footnote_suffix]
```

if it does have html in - if you mark the suffix as safe, concatenated them should leave them both mark as safe (assuming `super` returns an `html_safe` string)

Reply:
`String#+` is overridden by rails - if all components are html_safe, the string remains html_safe

```ruby
('a'.html_safe + 'b'.html_safe).html_safe? # => true

('a'.html_safe + 'b').html_safe? # => false
```